packages/@aws-cdk-testing/framework-integ/test/aws-batch/test/integ.ecs-job-definition.ts

```diff
@@ -60,6 +60,8 @@ new batch.EcsJobDefinition(stack, 'ECSFargateJobDefn', {
     memory: Size.mebibytes(32768),
     ephemeralStorageSize: Size.gibibytes(100),
     fargatePlatformVersion: FargatePlatformVersion.LATEST,
+    fargateCpuArchitecture: ecs.CpuArchitecture.ARM64,
+    fargateOperatingSystemFamily: ecs.OperatingSystemFamily.LINUX,
```

Comment:
Is it guaranteed that the `batch` `EcsFaragetContainerDefintion` will always match `ecs`? I am wondering if there is a case where ECS needs some update and then batch needs to follow. Meaning there would be a time where batch doesn't support all types ECS supports.

I might be looking at this in the wrong way or have an miss understanding on how things connect together within CDK.

Reply:
I saw it was already using enum values from ECS definition so I didn't really think about it... Looking at the current values they're the same but I'm not sure if that's guaranteed or it's documented somewhere.
If we can generate all these values from the schema in the future, then we can be confident about the values here.